lib/spoom/sorbet/sigils.rb

```diff
@@ -0,0 +1,39 @@
+# typed: true
+# frozen_string_literal: true
+
+module Spoom
+  module Sorbet
+    module Sigils
+      extend T::Sig
+
+      VALID_STRICTNESS = ["ignore", "false", "true", "strict", "strong"].freeze
+      SIGIL_REGEXP = /^#\s*typed\s*:\s*(\w*)$/.freeze
+
+      # returns the full sigil comment string for the passed strictness
+      sig { params(strictness: String).returns(String) }
+      def self.sigil(strictness)
+        "# typed: #{strictness}"
+      end
+
+      # returns true if the passed string is a sigil with valid strictness (else false)
+      sig { params(content: String).returns(T::Boolean) }
+      def self.valid_strictness?(content)
```

Comment:
If I understand correctly, in this module the semantics are:
* `sigil`: the full `# typed: strictness` comment
* `strictness`: the value of the strictness, ie. `true`, `false`...

Since this method takes the full `sigil` as input, shouldn't be called `valid_sigil?`?

Reply:
I don't think we need the checking services anymore since we only select the files with the relevant sigil.

The checking can then be delegated to `rubocop`.

Reply:
I've changed the method to accept a strictness string and it checks that it's valid. It's used in the initial check on the `--from` and `--to` arguments to the bump command. I'll hold off on any other checks that we don't need now.